CoolCurrency/UserAlerts.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  UserAlerts.swift
+//  CoolCurrency
+//
+//  Created by Vincent Moyo on 2021/11/16.
+//
+
+import Foundation
+import UIKit
+
+struct Alerts {
+    
+    static let updateFirstNameMessage = "Set your First Name to complete your profile account setup"
+    static let updateLastNameMessage = "Set your Last Name to complete your profile account setup"
+    static let resetEmailMessage = "Reset your email and insert a new one"
+    static let updateFirstNameTitle = "Set First name"
+    static let updateLastNameTitle = "Set Last  name"
+    static let resetEmailTitle = "Reset Email"
+    
+    static func showUserSuccessAlertExtension(for viewController: UIViewController,
+                                              forAnswer isCorrectAnswer: Bool,
+                                              title: String,
+                                              message: String,
+                                              action: @escaping ((UIAlertAction) -> Void)) {
+        let alert = UIAlertController(title: title,
+                                      message: message,
+                                      preferredStyle: .alert)
+        
+        alert.addAction(UIAlertAction(title: "OK",
+                                      style: .default,
+                                      handler: action))
+        
+        viewController.present(alert, animated: true, completion: nil)
+    }
+    
+    static func showUpdateFirstNameAlert(for viewController: UIViewController,
```

Comment:
So this are still a little bit too hard-coded, if I want to show a different alert I sort of need to come and add another function here, rather see if you can make one function that can handle these scenarios even if you have to pass through a few parameters 😄